stm32-modules/heater-shaker/firmware/motor_task/motor_policy.cpp

```diff
@@ -99,3 +101,16 @@ auto MotorPolicy::plate_lock_set_power(float power) -> void {
 auto MotorPolicy::plate_lock_disable() -> void {
     motor_hardware_plate_lock_off(&hw_handles->tim3);
 }
+
+auto MotorPolicy::set_pid_constants(double kp, double ki, double kd) -> void {
+    // These conversions match those in drive_parameters.h and therefore let you
+    // just look at the numeric literals there
+    static constexpr const double SPEED_UNIT_CONVERSION_SPC =
```

Comment:
Why are both `constexpr` and `const` used here?

Reply:
they imply different things annoyingly. Honestly it's probably fine to leave either or both of them off since compilers are pretty good about recognizing it but might as well slot em in there